readme.txt

```diff
@@ -20,8 +21,47 @@ For more information, see:
 * https://www.classicpress.net
 * https://github.com/ClassicPress/ClassicPress-Migration-Plugin
 
+== Installation ==
+
+To install a fresh version of ClassicPress:
+
+To install ClassicPress on a current WordPress site (switch a current WordPress site to ClassicPress):
+
+1. Backup the current site. You can do a manual backup in your hosting panel and export the database or you can use a backup plugin to do so. **PLEASE NOTE:** As ClassicPress is currently in Alpha, we do not recommend that you switch a live production site to ClassicPress.
+2. Download the ClassicPress Migration plugin from this link:  Insert Link
+3. Upload the plugin to the current WordPress plugins section of your site.
+4. Activate the ClassicPress Migration Plugin.
+5. Go to "Settings" of the ClassicPress Migration plugin at Tools->Switch to ClassicPress. All pre-flight checks must be marked in green to proceed with the switch.
+6. Press the "Switch To ClassicPress" button.
+7. This will take between one or two minutes so grab some water 🙂
+8. When the process is finished you should see the Welcome screen.
+
+If something is wrong please join our Slack group and ask in the #support channel.
+
+== Frequently Asked Questions ==
+
+= Will my current plugins and themes work in ClassicPress? =
+If your current plugins work in WordPress 4.9.x, they will work in ClassicPress too.  If you’re seeing something otherwise, that’s probably a bug with ClassicPress, and we’d appreciate you reporting it on Slack or GitHub.
+
+= I’m a developer, will I need to learn any new language or framework to develop in ClassicPress? =
+Not unless you want to!  We have some exciting features planned for version 2 and beyond, but they will all be optional and fully backwards-compatible.
+
+= I need help with something else, what should I do? =
+Like all of ClassicPress, our support is a volunteer effort by the community.  If you need help with something, please join our Slack group and ask in the #support channel.
+
+== Screenshots ==
+
+1. Screenshot 1 Description
+2. Screenshot 2 Description
+3. Screenshot 3 Description
+4. Screenshot 4 Description
+5. Screenshot 5 Description
+
 == Changelog ==
 
 = 0.0.1 =
```

Comment:
Now is a good time to bump the plugin version.  Let's go with `0.1.0` for now, and I'll change the rest of the places this appears in the code too.